### PR TITLE
docs: add comparation notes to the related projects

### PR DESCRIPTION
**Pull request by grikomsn**
## Description

This PR adds a "Comparison" section to explain differences between degit/tiged.

## Changes

- [feat(docs): add differences section](https://github.com/unjs/giget/commit/3d74a5128682692546fd44767a7c98a3cfc25d9b)
- [chore(docs): lint](https://github.com/unjs/giget/commit/b8cf0097d4b1df2102bd56cce1f9564137a6cb69)

## Notes

- We could add a short description about the section, but current attempt aims to be concise like other sections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,11 @@
 
 ✔ Support extracting with a subdir.
 
+## Comparison
+
+- Does not depend on local git to be installed ([read more](https://github.com/unjs/giget/issues/3#issuecomment-1240667190))
```
Comment (pi0) [inline README.md:26]: Would be better to make it a sentence directly in readme and also in [related projects](https://github.com/unjs/giget#related-projects) section. We shall move it above development

Comment (grikomsn) [reply README.md:26]: Does that mean we explain the tradeoffs and diffs in "related projects" section?

Comment (pi0) [reply README.md:26]: Yes i think that is more relavant section.

Comment (pi0) [reply README.md]: I've pushed some changes :)

**Comment by grikomsn** [comment]
Yours are much better. Sorry if it didn't seem helpful!

**Comment by pi0** [comment]
It was nice suggestion to add comparation! Please keep doing suggestions and contribution more :)

**Pull request merged**